blockchain/vm/jump_table.go

```diff
@@ -63,11 +63,23 @@ type operation struct {
 var (
 	ConstantinopleInstructionSet = newConstantinopleInstructionSet()
 	IstanbulInstructionSet       = newIstanbulInstructionSet()
+	LondonInstructionSet         = newLondonInstructionSet()
 )
 
 // JumpTable contains the EVM opcodes supported at a given fork.
 type JumpTable [256]*operation
 
+// newLondonInstructionSet returns the frontier, homestead, byzantium,
+// constantinople, istanbul, petersburg, berlin and london instructions.
```

Comment:
Do we have berlin instructions?

Reply:
Reflected at https://github.com/klaytn/klaytn/pull/1065/commits/ba5b68c1c296b02212a84c89da8cf446c3f9c7ee.

Reply:
@jeongkyun-oh I reverted the change, due to the definition of EVM compatible. Please check https://github.com/klaytn/klaytn/pull/1065/commits/e5e05f7f2341c6aa632ed37b0b05bf40b5b88bd4.
That is, `newLondonInstructionSet` is compatible with berlin HF, too.